scripts/lokalise/processor.rb

```diff
@@ -82,8 +82,9 @@ def determine_required_actions(local_keys)
             else
                 lokalise_entry = find_lokalise_entry_for_key(key_name)
                 is_missing_filename = lokalise_entry['filenames']['android'].empty?
+                needs_filename_update = lokalise_entry['filenames']['android'] != key[:filename]
```

Comment:
This will recognize if strings have been moved between modules, so we can just update `key[filenames][android]` on Lokalise.